src/locales/en.json

```diff
@@ -547,6 +547,7 @@
       "toggle-title": "PIN code",
       "toggle-description": "Will lock your app when switching apps or after 1m of inactivity."
     },
-    "fingerprint-text": "Or your fingerprint"
+    "fingerprint-text": "Or your fingerprint",
+    "use-fingerprint-question": "Do you want to use your fingerprint to unlock Banks ?"
```

Comment:
It seems to me that there is no space before punctuation in english. But I may be wrong

Reply:
Noice !